### PR TITLE
Feature/kitosudv it contract options

### PR DESCRIPTION
**Pull request by JacobMalling**
## KITOS Pull request template
The following procedure dictates the steps needed before a Pull request can be merged into master.

- [x] **Implement**: 
      *All requirements are implemented and unit tests are green*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*
      
- [x] **Request review**: 
      *Tag whomever you wish to review your code*
      
- [x] **Review completed**: 
      *Reviewer ticks this box when review comments have been submitted*
      
- [x] **Changes**: 
      *PR owner and reviewer agrees on which changes must be made and the changes are committed.*
      
- [x] **Merge master into branch / rebase with master**: 
      *Make sure you are testing your changes and how they co-exist with the latest version of master*
      
- [x] **Green on integration**: 
      *All integration tests are green on integration*

### CODE REVIEW

File: Presentation.Web/Controllers/API/V2/External/ItContracts/ItContractHandoverTrialTypeV2Controller.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+using System.Collections.Generic;
+using System.Net;
+using System.Web.Http;
+using Presentation.Web.Models.API.V2.Request.Generic.Queries;
+using Presentation.Web.Models.API.V2.Response.Generic.Identity;
+
+namespace Presentation.Web.Controllers.API.V2.External.ItContracts
+{
+    [RoutePrefix("api/v2/it-contract-handover-trial-types")]
+    public class ItContractHandoverTrialTypeV2Controller : BaseRegularOptionTypeV2Controller<HandoverTrial, HandoverTrialType>
```
Comment (JacobMalling) [inline Presentation.Web/Controllers/API/V2/External/ItContracts/ItContractHandoverTrialTypeV2Controller.cs:16]: Den her er lidt speciel. Her er roden ikke ItContract men derimod HandoverTrial.

File: Presentation.Web/Controllers/API/V2/External/ItContracts/ItContractAgreementElementTypeV2Controller.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        {
+        }
+
+        /// <summary>
+        /// Returns It-Contract agreement element type options 
+        /// </summary>
+        /// <param name="organizationUuid">organization context for the agreement element types availability</param>
+        /// <returns>A list of available It-Contract agreement element types</returns>
+        [HttpGet]
+        [Route("")]
```
Comment (mrjsawdk) [inline Presentation.Web/Controllers/API/V2/External/ItContracts/ItContractAgreementElementTypeV2Controller.cs:29]: Du behøves ikke `("")` (gælder også de andre controllers)

**Review by mrjsawdk — APPROVED** [review]
smukt.
En lille bemærkning

File: Presentation.Web/Tests/01_Parallel/data-processing/tabs/data-processing-registration.it-systems.e2e.spec.ts [hunk 3]
```diff
@@ -39,8 +37,9 @@ describe("Data processing registration it-systems test", () => {
     it("Assigning and removing systems",
         () => {
             const dprName = createName(3);
-
-            dpaHelper.createAndOpenDataProcessingRegistration(dprName)
+            SystemCatalogHelper.createSystemAndActivateLocally(system1)
+                .then(() => SystemCatalogHelper.createSystemAndActivateLocally(system2))
+                .then(() => dpaHelper.createAndOpenDataProcessingRegistration(dprName))
```
Comment (mrjsawdk) [inline Presentation.Web/Tests/01_Parallel/data-processing/tabs/data-processing-registration.it-systems.e2e.spec.ts:42]: måske man lige skulle huske at logge ud og ind som lokal admin inden selve scenariet udføres... vi skal undgå at gennemføre lokale tests som global admin for de har det med at skjule fejl (global admin er jo gud)

**Pull request merged**